src/components/LandingHero/index.tsx

```diff
@@ -42,28 +42,29 @@ export const LandingHero: FC = () => (
             <strong className='text-green'>
```

Comment:
Above there should be "Stadtpuls is **die** offene Datenplattform für IoT-Projekte Deiner Stadt"

Reply:
Done!